CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+- [#13089](https://github.com/MetaMask/metamask-extension/issues/13089): Active State for Lock Button is Incorrect Color.
```

Comment:
Thank you for the contribution @michaelklem !  You can omit this `changelog.md` entry, as we'll update this when it comes time for a release.  Thank you!

Reply:
Reverted my edits to this file.